go_lib/hooks/tls_certificate/internal_tls.go

```diff
@@ -224,10 +236,7 @@ func genSelfSignedTLS(conf GenSelfSignedTLSHookConf) func(input *go_hook.HookInp
 			}
 		}
 
-		// Note that []byte values will be encoded in base64. Use strings here!
-		input.Values.Set(caPath, cert.CA)
-		input.Values.Set(certPath, cert.Cert)
-		input.Values.Set(keyPath, cert.Key)
+		input.Values.Set(conf.path(), convCertToValues(cert))
```

Comment:
3 set actions seems to be more predictable then some structure, which could be changed. But, honestly, it's NIT dand if someone else has "pros"-position about this change - I don't mind